src/bench/bench.cpp

```diff
@@ -62,33 +88,39 @@ void benchmark::BenchRunner::RunAll(const Args& args)
     }
 
     std::vector<ankerl::nanobench::Result> benchmarkResults;
-    for (const auto& p : benchmarks()) {
-        if (!std::regex_match(p.first, baseMatch, reFilter)) {
+    for (const auto& [name, bench_func] : benchmarks()) {
+        const auto& [func, priority] = bench_func;
```

Comment:
nit: I think you renamed a bit too much - this one actually is a `priority_level`. My suggestion was to help distinguish between `PriorityLevel` and a `uint8_t`, not to just drop the "level" part everywhere. Same thing [here](https://github.com/bitcoin/bitcoin/compare/bacb1aef9c18d87cf6668f6ad057dbef4e7d9720..06664b8ef0c5eb85c63adb573724af0e3541933f#diff-6ab99df740dfacb06eb9c5897231240c2b7244d8c5d6e9de31b6912f9b4a5a69R67-R68) and [here](https://github.com/bitcoin/bitcoin/compare/bacb1aef9c18d87cf6668f6ad057dbef4e7d9720..06664b8ef0c5eb85c63adb573724af0e3541933f#diff-6ab99df740dfacb06eb9c5897231240c2b7244d8c5d6e9de31b6912f9b4a5a69R78-R81)

Reply:
pushed it.